docs/develop/node/running-a-node.md

```diff
@@ -188,3 +188,54 @@ To check logs call: `nearup logs` or `nearup logs --follow`
 >Got a question?
 <a href="https://stackoverflow.com/questions/tagged/nearprotocol">
   <h8>Ask it on StackOverflow!</h8></a>
+
+## Starting a node from backup
+There are public backups of data available as tar archives.
+This enables you to sync data from the network faster.
+We have regular backups and archival backups.
+We also keep those for two chains, mainnet and testnet.
```

Comment:
```suggestion
Using data backups allows you to sync your node quickly by using public tar backup files. There are two types of backups for available for both `testnet` and `mainnet`:
- regular 
- archival 
```

Reply:
Maybe we could add a description for regular and archival so people know what the difference between the two are.